api/elections.go

```diff
@@ -640,16 +640,17 @@ func getElection(electionID []byte, vs *state.State) (*models.Process, error) {
 func (a *API) electionFilterPaginatedHandler(msg *apirest.APIdata, ctx *httprouter.HTTPContext) error {
 	// get organizationId from the request body
 	body := struct {
-		OrganizationID types.HexBytes `json:"organizationId"`
-		ElectionID     types.HexBytes `json:"electionId"`
-		WithResults    bool           `json:"withResults"`
-		Status         string         `json:"status"`
+		OrganizationID types.HexBytes `json:"organizationId,omitempty"`
```

Comment:
I think this might be an abuse of struct types defined inside a function scope. IMO that makes sense for a very simple case (such as returning a json body with a single `Count` field). But for defining bigger structs, we have the api_types.go file.